pkg/internal/export/otel/common.go

```diff
@@ -55,6 +56,12 @@ var DefaultBuckets = Buckets{
 	RequestSizeHistogram: []float64{0, 32, 64, 128, 256, 512, 1024, 2048, 4096, 8192},
 }
 
+// Default values for the histogram configuration
+// From https://grafana.com/docs/mimir/latest/send/native-histograms/#migrate-from-classic-histograms
+var DefaultHistogramBucketFactor = 1.1
+var DefaultHistogramMaxBucketNumber = uint32(100)
+var DefaultHistogramMinResetDuration = 1 * time.Hour
```

Comment:
Should we allow users overriding the default histogram configuration?

Reply:
I think we could, but defeats the purpose a bit as the idea is users should worry about configuring buckets or any params for the histograms. But I can change it if you think is needed.

Reply:
Ok, let's keep it like now and allow overriding it in the future, only if we feel the need for it.